app/models/campus_oracle/queries.rb

```diff
@@ -38,7 +38,7 @@ def self.find_people_by_name(name_search_string, limit = 0)
       self.raise_non_test_error
 
       raise ArgumentError, "Search text argument must be a string" if name_search_string.class != String
-      raise ArgumentError, "Limit argument must be a Fixnum" if limit.class != Fixnum
+      raise ArgumentError, "Limit argument must be a Integer" if limit.class != Integer
```

Comment:
See [Ruby 2.4 unifies Fixnum and Bignum into Integer](https://blog.bigbinary.com/2016/11/18/ruby-2-4-unifies-fixnum-and-bignum-into-integer.html)